### PR TITLE
lrr.in.tum.de rule

### PR DESCRIPTION
**Pull request by jbreitbart**

### CODE REVIEW

**Comment by jbreitbart** [comment]
Can someone explain to me why one of the travis tests fails? I don't see anything useful in the error log at https://travis-ci.org/EFForg/https-everywhere/jobs/136743355

**Comment by J0WI** [comment]
We tend to always cover the whole *.tum.de host, not just a single subdomain.
Would you like to add others too? You can find a guide here:
https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md

**Comment by jbreitbart** [comment]
I'm afraid, I cannot make any rule for the whole domain, there are too many subdomains that are not handled in a uniform way.

Please close the issue if you prefer not to have single subdomain rules.

**Comment by J0WI** [comment]
We already have `Technical_University_Munich.xml` and `Tum.de.xml` (which seems obsolete anyway). You can add this host there.

**Comment by jbreitbart** [comment]
@J0WI fixed + added some tests. Some of the certificates for TUM have expired and the tests fail. What is the best solution for this?

**Comment by J0WI** [comment]
The best is to remove the targets/redirect rules for them and add comments on the top about their issue.

It would also be nice if you could remove the wildcard targets and replace them with the hosts that are supporting https. This would also avoid test urls and non-ca-capturing groups.

**Comment by jbreitbart** [comment]
@J0WI Fixed. Let me know if there are any more issues. Also feel-free to squash-merge.

File: src/chrome/content/rules/Technical_University_Munich.xml [hunk 1]
```diff
@@ -45,37 +39,68 @@
...
 
 	<!--	Not secured by server:
 					-->
 	<!--securecookie host="^www\.sec\.in\.tum\.de$" name="^PHPSESSID$" /-->
 	<!--securecookie host="^mediatum\.ub\.tum\.de$" name="^PSESSION$" /-->
 
 	<securecookie host="^(?:in|(?:adm9|ias|sec|www)\.in|mediatum\.ub|www(?:\.lehren?)?)?\.tum\.de$" name=".+" />
-	<securecookie host="^ias\.cs\.tum\.edu$" name=".+" />
-	<securecookie host="^(?:www|ias\.informatik)\.tu-muenchen\.de$" name=".+" />
+	<securecookie host="^(?:www\.informatik)\.tu-muenchen\.de$" name=".+" />
```
Comment (J0WI) [inline src/chrome/content/rules/Technical_University_Munich.xml]: Please also add `www.informatik.tu-muenchen.de` as target host and replace all non-capturing groups in this ruleset (`(?:www\.informatik)`) with normal ones (`(www\.informatik)?`).

File: src/chrome/content/rules/Tum.de.xml [hunk 2]
```diff
@@ -1,10 +0,0 @@
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://tum.de/ => https://www.tum.de/: (6, 'Could not resolve host: tum.de')
--->
-<ruleset name="Tum.de" default_off='failed ruleset test'>
-  <target host="tum.de" />
```
Comment (J0WI) [inline src/chrome/content/rules/Tum.de.xml]: You should give a hint in the comments on the top of Technical_University_Munich.xml  that ^tum.de and ^tu-muenchen.de do not exist.

**Comment by J0WI** [comment]
Please use a generic `<rule from="^http:" to="https:" />` to just redirecting all listed targets to https.

**Comment by jeremyn** [comment]
@jbreitbart Are you still interested in working on this?

File: src/chrome/content/rules/Tum.de.xml (hunk 2 above)
Comment (jbreitbart) [inline src/chrome/content/rules/Tum.de.xml]: See line 21

File: src/chrome/content/rules/Technical_University_Munich.xml (hunk 1 above)
Comment (jbreitbart) [inline src/chrome/content/rules/Technical_University_Munich.xml]: I added the target host, but I don't know how to remove the non-capturing groups. Sorry.

**Comment by jbreitbart** [comment]
@jeremyn Thanks for the reminder.

I rebased my fork on the current master and updated it. See comments above.

**Comment by jeremyn** [comment]
Using Sublist3r (see the [style guide](https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md)) I see there are many, many more `tum.de` domains than what you've listed here. Do you mean in https://github.com/EFForg/https-everywhere/pull/5084#issuecomment-242321518 that you're not interested in adding any more domains than what are already included?

**Comment by jeremyn** [comment]
@jbreitbart I'm trying to figure out what to do with this pull request. Are you interested in adding the missing `tum.de` subdomains?

**Comment by jeremyn** [comment]
Hi @jbreitbart, are you still interested in working on this pull request? If not, I'm going to close it.

**Comment by jeremyn** [comment]
No response, closing.

**Pull request closed without merging**